applications/luci-app-upnp/htdocs/luci-static/resources/view/upnp/upnp.js

```diff
@@ -153,18 +153,18 @@ return view.extend({
 		o.datatype    = 'uinteger'
 		o.placeholder = 30
 
-		o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Clean rules threshold'))
+		o = s.taboption('advanced', form.Value, 'clean_ruleset_threshold', _('Clean rules threshold of number of rules cleaned once'))
 		o.datatype    = 'uinteger'
 		o.placeholder = 20
 
-		o = s.taboption('advanced', form.Value, 'clean_ruleset_interval', _('Clean rules interval'))
+		o = s.taboption('advanced', form.Value, 'clean_ruleset_interval', _('Clean rules (delay) interval'))
```

Comment:
It's not a delay but an interval.

Reply:
Yeah, let's reword to "Intra-cleanse interval". And a subtext like ("%s means immediate").format("<code>0</code>");

Reply:
```js
		o = s.taboption('advanced', form.Value, 'clean_ruleset_interval', _('Interval between cleanse runs'), _('Unit: seconds. %s means disable').format("<code>0</code>"));
```